pkgs/tools/misc/android-tools/default.nix

```diff
@@ -23,8 +27,13 @@ stdenv.mkDerivation rec {
     })
   ];
 
+  postPatch = ''
+    sed -i -E "0,/import api_pb2/ s//from google.protobuf import api_pb2/" vendor/avb/aftltool.py
```

Comment:
Could we switch to `substituteInPlace` here? That way it emits a warning if this doesn't match anymore.

Reply:
I used `substituteInPlace` first but there is exact two `import api_pb2` and we only need to fix first one. Check [this line](https://android.googlesource.com/platform/external/avb/+/4749d5547d9f5f54b0407d82b5f52ea85c8f0d9c/aftltool.py#1497). `substituteInPlace` replaces both of them.

Reply:
Ok, that also explains the sed expression then, thanks! Using e.g. `s/^import api_pb2$/from google.protobuf import api_pb2/` should also work then but it's fine either way.